spec/plsql/schema_spec.rb

```diff
@@ -86,7 +86,7 @@
   it "should not connect with wrong port number" do
     expect {
       plsql.connect! @username, @password, :host => @host, :port => 9999, :database => @database
-    }.to raise_error(/no listener|could not establish the connection/)
+    }.to raise_error(/ORA-12541/)
```

Comment:
> In fact, now that you mention it, it think it would be better not to set the ENV parameters at all, or set them to some defaults, if they are not set.
> That should go for:
> NLS_LANG Oracle session language
> ORA_SDTZ Oracle session timezone 
> TZ ruby timezone parameter
> So by default you would get:
> 
>> TZ=UTC
>> ORA_SDTZ=UTC
>> NLS_LANG=AMERICAN_AMERICA.AL32UTF8
> Anyway, users should be able to change those settings as they need, to setup any valid language and timezone on their environment.
> I can update the spec_helper.rb to reflect that.
> Would that be OK?

* Sounds OK at first glimpse. I'm not sure about the TZ=UTC, maybe it would be better to use e.g. `Europe/Riga` as default - otherwise tests won't be able to capture a buggy DST-sensitive behavior.
* Let's say a user wants to save the local custom settings (to avoid typing them each time): what would be the best: a .gitignored file (like the idea with `custom_settings.rb`); or some external script which to remember to invoke (we anyway may need that for ORACLE_HOME, PATH, etc.)

Maybe it's best to keep customization (of TZ, etc) scope of impact as small as possible - e.g. for ruby-plsql rspec tests only, but not for my shell. For my shell - I may have my own preferences for local time, language, and so on.